lando/server/jobapi.py

```diff
@@ -0,0 +1,274 @@
+"""
+Allows reading and updating job information when talking to Bespin REST api.
+"""
+from __future__ import print_function
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+class BespinApi(object):
+    """
+    Low level api that interfaces with the bespin REST api.
+    """
+    def __init__(self, config):
+        """
+        :param config: ServerConfig: contains settings for connecting to REST api
+        """
+        self.settings = config.job_api_settings
+        self.requests = requests # Allows mocking requests for unit testing
+
+    def auth(self):
+        """
+        Create http auth based on config passed in constructor.
+        :return: HTTPBasicAuth
+        """
+        return HTTPBasicAuth(self.settings.username, self.settings.password)
```

Comment:
This is not documented well in the bespin-api project, but we have [TokenAuthentication](http://www.django-rest-framework.org/api-guide/authentication/#tokenauthentication) enabled if that's preferred over username/password: [bespin-api/bespin/settings.py](https://github.com/Duke-GCB/bespin-api/blob/eb23233f785365cb16672933011eed3fd90ac0ac/bespin/settings.py#L139).